Makefile

```
@@ -0,0 +1,2 @@
+all:
+	docker-compose build --build-arg USER_ID=$(shell id -u) --build-arg GROUP_ID=$(shell id -g)
```

The build argument `GROUP_ID` isn't used in the Dockerfile. We can remove it.

```suggestion
	docker-compose build --build-arg USER_ID=$(shell id -u)
```

---

In the end, @vpereira and I talked and it's better to pass the group ID and use it in the Dockerfile instead of harcoding the user group to `users`.